evm_loader/program/src/executor/state.rs

```diff
@@ -285,17 +285,39 @@ impl<'a, B: AccountStorage> Database for ExecutorState<'a, B> {
     }
 
     async fn code_hash(&self, from_address: Address, chain_id: u64) -> Result<[u8; 32]> {
```

Comment:
I suggest renaming `from_address` parameter to `address` to make it consistent with the trait method definition.